src/appengine/requirements.txt

```diff
@@ -0,0 +1,8 @@
+firebase-admin==2.16.0
+Jinja2==2.11.1
+jira==2.0.0
+PyJWT==1.7.1
+pyOpenSSL==19.1.0
```

Comment:
Isnt pyOpenSSL coming from requirements.txt.

Reply:
The only other place this was getting installed was on bots (which is platform specific). We can maybe even get rid of that there.